model/converter/json/fixtures/ui_01.json

```diff
@@ -62,6 +62,11 @@
           "type": "float64",
           "value": 72.5
         },
+        {
+          "key":"javascript_limit",
+          "vType":"INT64",
+          "vInt64":9223372036854775222
```

Comment:
Yes it was shown as a string in ui_01-actual.json . There is a type mismatch between domain_01.json (int64) and ui_01-actual.json (string) and hence the test case TestFromDomain was failing

Reply:
I am not talking about `vType` field, it should indeed remain `INT64`, but the value of `vInt64` field needs to be a string, not a raw number, because the raw number is what's causing the precision loss.